lib/Db/LocalAttachmentMapper.php

```diff
@@ -37,15 +37,20 @@ public function __construct(IDBConnection $db) {
 
 	/**
 	 * @throws DoesNotExistException
+	 *
 	 * @param int $userId
 	 * @param int $id
+	 *
 	 * @return LocalAttachment
 	 */
 	public function find($userId, $id) {
-		$sql = 'SELECT * FROM `' . $this->getTableName() . '` WHERE user_id = ? and id = ?';
-		$params = [$userId, $id];
+		$qb = $this->db->getQueryBuilder();
+		$query = $qb
+			->select('*')
+			->from($this->getTableName())
+			->where($qb->expr()->eq('user_id', $qb->createNamedParameter($userId)));
```

Comment:
The oroginal statement has `id = ?` this one not. Intentional?

Reply:
Nope, my fault.